website_product/tests/__init__.py

```diff
@@ -0,0 +1 @@
+from . import test_website_product
```

Comment:
I don't care that much but.... linters will complain about not having this new line here